The Eclipselink download page contains OSGi bundles with SNAPSHOT versions

**Describe the bug**

The page https://eclipse.dev/eclipselink/#download contains links to "EclipseLink OSGi Bundles Zip" for each released version. All the Eclipselink JAR files in them contain a MANIFEST.MF file with a SNAPSHOT OSGi bundle version, e.g. `Bundle-Version: 4.0.4-SNAPSHOT.v202407151019`

This causes Felix, e.g. in Payara or GlassFish, fail to load the bundles because it expects a version only with numbers and periods, no SNAPSHOT in the version. See this question: https://stackoverflow.com/questions/78673772/how-to-upgrade-eclipselink-in-payara-server?noredirect=1#comment138942192_78673772

**To Reproduce**

- EclipseLink version: 4.0.4, 4.0.3, 4.0.2, possibly older versions too
- Java/JDK version: any

Try to load any of those bundles in a Felix OSGi container. It will fail with:

```
org.osgi.framework.BundleException: Could not create bundle object.
at org.apache.felix.framework.Felix.installBundle(Felix.java:3339)
...
Caused by: java.lang.IllegalArgumentException: invalid version "4.0.3- SNAPSHOT.v202405161415": non-numeric "3-SNAPSHOT"
...
Caused by: java.lang.NumberFormatException: For input string: "3-SNAPSHOT"
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
```

**Expected behavior**
All the distributed OSGi bundles should have a non-SNAPSHOT version.

**Additional context**
The artifacts released to Maven Central have the correct version without SNAPSHOT: https://search.maven.org/search?q=org.eclipse.persistence

--- a/pom.xml
+++ b/pom.xml
@@ -120,7 +120,7 @@
         <build.type>SNAPSHOT</build.type>
         <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
         <build.qualifier>v${maven.build.timestamp}</build.qualifier>
-        <bundle.version>${project.version}.${build.qualifier}</bundle.version>
+        <bundle.version>${release.version}.${build.qualifier}</bundle.version>
         <project.build.testReports.settings>test-report-settings/surefire-report.properties</project.build.testReports.settings>
         <project.build.testReports.subdirectory>test-reports</project.build.testReports.subdirectory>
         <project.build.testReports.summaryFile>test-summary</project.build.testReports.summaryFile>
@@ -1664,6 +1664,7 @@
                         <_noextraheaders>true</_noextraheaders>
                         <_noimportjava>true</_noimportjava>
                         <HK2-Bundle-Name>${project.groupId}:${project.artifactId}</HK2-Bundle-Name>
+                        <Bundle-Version>${bundle.version}</Bundle-Version>
                     </instructions>
                 </configuration>
                 <executions>

